testing/sdk_cmd.py

```diff
@@ -27,6 +29,6 @@ def run_cli(cmd, print_output=True):
     if ret != 0:
         err = 'Got error code {} when running command "dcos {}":\nstdout: "{}"\nstderr: "{}"'.format(
             ret, cmd, stdout, stderr)
-        sdk_utils.out(err)
+        log.info(err)
```

Comment:
This PR was simply intended just to cover the unification but I'm glad to add whatever levels you care to suggest